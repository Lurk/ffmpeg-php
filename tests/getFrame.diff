001+ Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/getFrame.php on line 8
001- ffmpeg getFrame(): md5 = f8eb6d231d353e8d98cf76f1fcf960c3