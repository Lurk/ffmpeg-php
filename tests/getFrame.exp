ffmpeg getFrame(): md5 = f8eb6d231d353e8d98cf76f1fcf960c3