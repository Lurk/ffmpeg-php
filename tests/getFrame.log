
---- EXPECTED OUTPUT
ffmpeg getFrame(): md5 = f8eb6d231d353e8d98cf76f1fcf960c3
---- ACTUAL OUTPUT
Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/getFrame.php on line 8
---- FAILED
