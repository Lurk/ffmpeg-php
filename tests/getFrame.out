Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/getFrame.php on line 8